ui/lib/core/addon/components/alert-banner.hbs

```diff
@@ -30,6 +30,12 @@
             {{yield}}
           </p>
         {{/if}}
```

Comment:
This is the only addition to the alert-banner component. If the user passes in a learnLink then it will show the learn-link icon and a message they must also pass in.